stories/Socials/Socials.stories.jsx

```diff
@@ -0,0 +1,45 @@
+import Socials from '../../components/Socials';
+
+const Template = args => <Socials {...args} />;
+
+const allItems = [
+  {
+    name: 'facebook',
+    url: 'https://facebook.com',
+    description: 'facebook description of the purpose of this icon',
+  },
+  {
+    name: 'twitter',
+    url: 'https://twitter.com',
+    description: 'twitter description of the purpose of this icon',
+  },
+  {
+    name: 'youtube',
+    url: 'https://youtube.com',
+    description: 'youtube description of the purpose of this icon',
+  },
+];
+
+export const AllItems = Template.bind({});
+AllItems.args = {
+  items: allItems,
+};
```

Comment:
```suggestion
```
When putting a default value in the template, this argument is not needed in this example